pkg/provisioner/templates/kubernetes.go

```diff
@@ -147,12 +153,116 @@ echo "you can now access the cluster with:"
 echo "ssh -i <your-private-key> ubuntu@${INSTANCE_ENDPOINT_HOST}"
 `
 
+const NVKindTemplate = `
```

Comment:
Looking at this template, it seems as if we're duplicating the logic to install kind. Can we not have `kind` as a prerequisite for `nvkind`?

Reply:
resolved